agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/exporter/Exporter.java

```diff
@@ -129,20 +99,55 @@
   private static final AttributeKey<String> AI_LOG_ERROR_STACK_KEY =
       AttributeKey.stringKey("applicationinsights.internal.log_error_stack");
 
-  // note: this gets filtered out of user dimensions automatically since it shares official "peer."
-  // prefix
-  // (even though it's not an official semantic convention attribute)
+  private static final AttributeKey<String> AZURE_NAMESPACE =
+      AttributeKey.stringKey("az.namespace");
   private static final AttributeKey<String> AZURE_SDK_PEER_ADDRESS =
       AttributeKey.stringKey("peer.address");
   private static final AttributeKey<String> AZURE_SDK_MESSAGE_BUS_DESTINATION =
       AttributeKey.stringKey("message_bus.destination");
+  private static final AttributeKey<Long> AZURE_SDK_ENQUEUED_TIME =
+      AttributeKey.longKey("x-opt-enqueued-time");
 
   private static final OperationLogger exportingSpanLogger =
       new OperationLogger(Exporter.class, "Exporting span");
 
   private static final OperationLogger parsingHttpUrlLogger =
       new OperationLogger(Exporter.class, "Parsing http.url");
 
+  static {
+    Set<String> dbSystems = new HashSet<>();
```

Comment:
does otel java sdk have dbSystems defined already? can we refer to that directly if it becomes available? i remember there is a common spec for dbsystems on the otel spec repo.

Reply:
👍